lib/sinatra/slack/instance_helpers.rb

```diff
@@ -27,7 +27,7 @@ def slack_response(callback_id)
         s_resp
       end
 
-      def slack_error_notification
+      def slack_error_notification(error)
```

Comment:
You added this `error` parameter, but you are not using it. Something that you forgot to remove or commit?

Reply:
No, the idea here is to allow `slack_error_notification` to be overridden, if necessary. 

Without this argument here, if the user doesn't override this method an exception is thrown, since we'd be sending an argument to a method that doesn't allow any arguments.

P.S. I apologize if this is not the best way to do this, but I'm not very proficient in Ruby...

Reply:
Oh right. Makes sense. It's ok, then. I would only add the `_` before the name. It's a best practice. It would become something like this:
```suggestion
      def slack_error_notification(_error)
```